spec/order_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'order'
+
+describe Order do 
+
+  it { is_expected.to respond_to(:add_item).with(2).arguments }
+  it { is_expected.to respond_to(:remove_item).with(2).argument }
+  it { is_expected.to respond_to(:check_dishes) }
+  it { is_expected.to respond_to(:basket) }
+  it { is_expected.to respond_to(:place_order) }
+  
+
+  it 'should be able to check the list of dishes with prices' do
+    expect(subject.check_dishes).to be_truthy
+  end 
+
+  it 'should have the initial total as 0' do 
+    expect(subject.total).to eq(0.0)
+  end
+
+  context 'Adding items' do 
+
+    it 'should be able to add a dish to the basket' do
+      subject.add_item("Duck Pancakes")
+      expect(subject.basket.include?("Duck Pancakes")).to be(true)
+    end
+
+    it 'should increase the total price correctly after adding an item' do
+      subject.add_item("Duck Pancakes")
+      expect(subject.total).to eq(4.99)
+    end
+
+    it 'should not be able to add items not on the menu' do 
+      expect{ subject.add_item("Chicken Korma") }.to raise_error 
+        'That item is not on the menu'
+    end
+
+    it 'should add to the quantity if the same item is added more than once' do
+      subject.add_item("Duck Pancakes")
+      subject.add_item("Duck Pancakes", 3)
+      expect(subject.basket["Duck Pancakes"]).to eq(4)
+    end
+
+  end
+
+  context 'Removing items' do
+
+    it 'should be able to remove an item from the basket' do
+      subject.add_item("Duck Pancakes")
+      subject.remove_item("Duck Pancakes")
+      expect(subject.basket.empty?).to eq(true)
+    end
+
+    it 'should not be able to remove an item not in the basket' do
+      subject.add_item("Duck Pancakes")
+      expect{ subject.remove_item("Egg Fried Rice") }.to raise_error 
+        'Item was not in the basket'
```

Comment:
When you add a line break to put the error message on the next line, you're dissociating the error message you are wanting to compare against. Your test still passes, but only because ANY kind of error is raised. Did you notice a kind of 'false positive' warning when you ran the tests? I suspect this was an attempt to stop hound barking about long line lengths but in this case the workaround breaks the test slightly.